exporter/signalfxexporter/translation/constants.go

```diff
@@ -138,6 +138,37 @@ translation_rules:
     receive: pod_network_receive_errors_total
     transmit: pod_network_transmit_errors_total
 
+# cpu deltas
+- action: delta_metric
+  mapping:
+    system.cpu.time: system.cpu.delta
+- action: copy_metrics
+  mapping:
+    system.cpu.delta: system.cpu.usage
+  dimension_key: state
+  dimension_values:
+    user: true
+    system: true
+    interrupt: true
+- action: aggregate_metric
+  metric_name: system.cpu.usage
+  aggregation_method: sum
+  without_dimensions:
+  - state
+- action: copy_metrics
+  mapping:
+    system.cpu.delta: system.cpu.total
+- action: aggregate_metric
+  metric_name: system.cpu.total
+  aggregation_method: sum
+  without_dimensions:
+  - state
+- action: calculate_new_metric
+  metric_name: cpu.utilization
+  operand1_metric: system.cpu.usage
+  operand2_metric: system.cpu.total
+  operator: /
```

Comment:
let's remove `system.cpu.usage` and `system.cpu.total` after that

Reply:
Done. Removed `system.cpu.delta` as well.